simulator/src/plotArea.js

```
@@ -432,12 +432,26 @@ export function setupTimingListeners() {
     })
-    $('.timing-diagram-resume').on('click', () => {
+    const ResumePause = document.querySelector('.ResumePause');
+    let resumeAutoscroll = false;
+
+    function resumeTimingDiagram() {
+        resumeAutoscroll = true;
         plotArea.resume();
-    })
-    $('.timing-diagram-pause').on('click', () => {
+        ResumePause.innerHTML = '<span class="fas fa-pause timing-diagram-pause">';
+        ResumePause.title='Pause auto-scroll';
```

ESLint

[error] Operator '=' must be spaced.

(space-infix-ops)